configure

```diff
@@ -2811,10 +2811,12 @@ case $_host_os in
 		append_var CXXFLAGS "-fmodulo-sched"
 		append_var CXXFLAGS "-fuse-cxa-atexit"
 		append_var CXXFLAGS "-I$DEVKITPRO/libogc/include"
+		append_var CXXFLAGS "-isystem $DEVKITPRO/portlibs/ppc/include"
```

Comment:
does dkppc use `-isystem` as well? (compare to just `-I`)

Reply:
Good catch!
[devkitPro](https://github.com/devkitPro/pacman-packages/blob/master/pkgbuild-scripts/ppcvars.sh) seems to use -I.
I will correct that.

Reply:
Corrected.